doc/rst/source/changes.rst

```diff
@@ -4,6 +4,83 @@
 Changelog
 =========
 
+New Features in GMT 6.1
+=======================
+
+GMT 6.1 may be a minor revision to 6.0 but packs quite a punch. For general
+changes, we mention
+
+ #. Updated remote global data sets: Earth relief, land/ocean masks, and day/night imagery. The larger
+    grid files (5x5 arc minutes and smaller resolutions) are now tiled and faster to download.
+ #. Add MVT (Mapbox Vector Tile) as another GDAL URL dataset.
```

Comment:
Better remove this one. We do nothing with the vector tiles yet. It was just to anticipate future use.

Reply:
done